src/data/rat/basic.lean

```diff
@@ -744,6 +744,26 @@ begin
     coe_nat_div_self]
 end
 
+lemma num_inv_nat {a : ℕ} (ha0 : 0 < a) : (a : ℚ)⁻¹.num = 1 :=
```

Comment:
I think this should maybe be called `inv_coe_nat_num` to match `coe_nat_num`. Does that seem ok to you?

Reply:
Yes that makes sense!  Will change both names.